action.yml

```diff
@@ -26,6 +26,11 @@ inputs:
   config-file-path:
     description: Path to the config file for the license checker.
     required: true
+outputs:
+  notice-file-path:
+    description: Path to the generated notice file
+  notice-file-is-dirty:
+    description: True, if the generated notice file "is dirty" (needs being updated in the git repo)
```

Comment:
Spelling suggestion: "True if the notice file has been updated and needs to be commited via git, False otherwise."